src/components/ui/Badge/Badge.tsx

```diff
@@ -0,0 +1,27 @@
+import { Badge as UIBadge } from '@faststore/ui'
+import type { ReactNode } from 'react'
+import React from 'react'
+
+import './badge.scss'
+
+export type BadgeVariants = 'outOfStock' | 'new' | 'recommended' | 'discount'
+
+type Props = {
+  small?: boolean
+  variant: BadgeVariants
+  children: ReactNode
+}
+
+const Badge = ({ small = false, variant, children }: Props) => {
```

Comment:
I'm rethinking about this component. Maybe we can just need this `Badge`. I'm not sure if we still need `DiscountBadge` and `ProductBadge`. Is there something special that can justify these other components? 🤔

Reply:
The discount badge encapsulate the calc of discount logic and the Product Badge the labels of the others badges. How you`re thinking this component?